.todir

```diff
@@ -0,0 +1 @@
+# Add bower installer inside dockerfile
```

Comment:
Please refrain from committing `.todir` files. I know it is useful for you, but a GitHub Issue can be more appropriate. That way we can participate in a discussion, track the progress, etc.

Thanks!